frontend/src/app/core/interceptors/error.interceptor.ts

```diff
@@ -32,9 +32,7 @@ export class ErrorInterceptor implements HttpInterceptor {
     intercept(req: HttpRequest<unknown>, next: HttpHandler) {
         return next.handle(req).pipe(catchError(response => {
             if (response.status === 401) {
-                if (response.headers.has('Token-Expired')) {
-                    return from(this.handleExpired(req, next));
-                }
+                return from(this.handleExpired(req, next));
```

Comment:
why delete response.headers.has('Token-Expired') ? we will update token for all cases with 401 error ?

Reply:
Because we don't get this header in the response. Also it allows getting correct token in case of accidental or not modification of token in the local storage

Reply:
![Screenshot_669](https://user-images.githubusercontent.com/88149444/188008428-41dd1ee2-a52d-495c-8816-e26e5047ed25.png)